src/helpers/models.js

```diff
@@ -0,0 +1,55 @@
+import async from 'async';
+
+/**
+ * paginates sequelize model query
+ *
+ * @function
+ *
+ * @param {Function} sourceFunction - function to call as data source
+ * @param {Object} dataToFunction - data to pass to source function as parameter
+ * @param {number} size - size of data to return per page
+ *
+ * @return {Object} - sequelize object for inserted data
+ */
+export const QueryPagination = async (sourceFunction, dataToFunction = {}, size = 50) => {
```

Comment:
This is not a Class or a Model, it's a helper and should have a camelCased name. Please fix all functions in this file